### PR TITLE
Add a new section with IntelliJ tips & tricks

### PR DESCRIPTION
**Pull request by mit-mit**
A few tips for new IJ users

Explain code completion case sensitivity (Fixes https://github.com/flutter/flutter-intellij/issues/552)

### CODE REVIEW

**Comment by mit-mit** [comment]
@devoncarew @sethladd please review

**Comment by devoncarew** [comment]
👍 lgtm

File: intellij-ide.md [hunk 1]
```diff
@@ -185,6 +185,21 @@ your debugging session:
...
+  * JetBrains has a good [Keyboard Shortcuts You Cannot Miss](https://www.jetbrains.com/help/idea/keyboard-shortcuts-you-cannot-miss.html) overview
+  * Customize the shortcuts to your preferences in **Preferences > Keymap**
+
+To navigate quickly in your code:
+  * Switch between recently edited files with `cmd-e`
+  * Navigate between all files with `cmd-arrow_up`
+
+Code completion:
+  * Code completions will be shown as you type, or can be invoked at any time with `shift-spacebar`
+  * IntelliJ code completion matches are per-default case sensitive, even on the first letter.
```
Comment (alexander-doroshko) [inline intellij-ide.md]: @mit-mit 
by default - ONLY on the first letter :)
The rest is case insensitive by default.

File: intellij-ide.md [hunk 2]
```diff
@@ -185,6 +185,21 @@ your debugging session:
 1. In the Debug panel at the bottom, click the **Full Application Restart button**
 (shown with an icon containing a circular arrow).
 
+## Tips and tricks
+
+Keyboard shortcuts:
+  * JetBrains has a good [Keyboard Shortcuts You Cannot Miss](https://www.jetbrains.com/help/idea/keyboard-shortcuts-you-cannot-miss.html) overview
+  * Customize the shortcuts to your preferences in **Preferences > Keymap**
```
Comment (alexander-doroshko) [inline intellij-ide.md]: we usually recommend to learn the default shortcuts rather than customize them :)

File: intellij-ide.md [hunk 3]
```diff
@@ -185,6 +185,21 @@ your debugging session:
...
+
+To navigate quickly in your code:
+  * Switch between recently edited files with `cmd-e`
```
Comment (alexander-doroshko) [inline intellij-ide.md]: Cmd+E is Recently opened files,
Recently Edited Files is Cmd+Shift+E.
And on Linux (and Windows if ever supported) shortcuts are different :)

**Pull request merged**